### PR TITLE
fix link in successful publicize message

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform 
Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: cmd/publicize/server.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func (s *server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
 		return
 	}
 
-	destOrgRepoLink := fmt.Sprintf("https:///%s/tree/%s", destOrgRepo, baseBranch)
```
Comment (alvaroaleman) [inline cmd/publicize/server.go:105]: What about using tests to avoid this kind of issue?

Comment (droslean) [reply cmd/publicize/server.go:105]: I will add more tests.

**Pull request merged**